conf/messages.cy

```diff
@@ -469,32 +469,63 @@ pdf.ruling.aboutDecision.text=Rwyf wedi gwneud y penderfyniad hwn o dan adran 24
 pdf.ruling.coverLetter.title=Llythyr eglurhaol am y Dyfarniad Tariffau Uwch
+
 pdf.ruling.section.ifYouDisagree.title=Yr hyn i’w wneud os ydych yn anghytuno â’r penderfyniad hwn
 pdf.ruling.section.ifYouDisagree=Os ydych yn anghytuno â’r penderfyniad hwn, gallwch naill ai:
 pdf.ruling.section.ifYouDisagree.1=gofyn am adolygiad gan CThEF
 pdf.ruling.section.ifYouDisagree.2=apelio ar dribiwnlys annibynnol
-pdf.ruling.section.ifYouDisagree.cannot=Ni allwch ofyn i CThEF gynnal adolygiad ac apelio ar dribiwnlys annibynnol ar yr un pryd. Fodd bynnag, gallwch apelio ar dribiwnlys annibynnol os ydych yn anghytuno â chanlyniad eich adolygiad.
-pdf.ruling.section.ifYouDisagree.moreInfo=I gael rhagor o wybodaeth am adolygiadau ac apeliadau, gallwch
-pdf.ruling.section.ifYouDisagree.read=ddarllen dogfen HMRC1
-pdf.ruling.section.ifYouDisagree.phone=neu ffonio 0300 200 3705.
+pdf.ruling.section.ifYouDisagree.within=Mae'n rhaid i chi wneud hynny cyn pen 30 diwrnod o ddyddiad y llythyr hwn.
+pdf.ruling.section.ifYouDisagree.cannot=Ni allwch ofyn i CThEF gynnal adolygiad ac apelio ar dribiwnlys annibynnol ar yr un pryd. Fodd bynnag, gallwch apelio ar dribiwnlys annibynnol os ydych yn anghytuno â chanlyniad eich adolygiad. Os byddwch yn apelio ar dribiwnlys, ni ellir cynnig adolygiad.
+
 pdf.ruling.section.askForReview.title=Gofyn am adolygiad gan CThEF
-pdf.ruling.section.askForReview.email=E-bostiwch <strong>{0}</strong> yn gofyn am adolygiad a rhowch eich rhesymau cyn pen 30 diwrnod ar ôl cael yr e-bost am eich dyfarniad.
+pdf.ruling.section.askForReview.reviewOfficer=Bydd swyddog adolygu yn edrych ar y penderfyniad. Mae'r swyddog hwnnw yn ddiduedd. Bydd canlyniad yr adolygiad yn cael ei anfon atoch cyn pen 45 diwrnod.
+pdf.ruling.section.askForReview.ifYouDisagree=Os byddwch yn anghytuno â chanlyniad yr adolygiad, bydd yn dal i fod yn bosibl i chi apelio ar y tribiwnlys. Mae'n rhaid i chi wneud hyn cyn pen 30 diwrnod i ddyddiad y llythyr sy'n rhoi gwybod i chi am ganlyniad yr adolygiad. Ar ôl i adolygiad cael ei ddechrau, ni allwch apelio ar dribiwnlys hyd nes y bydd y swyddog adolygu'n rhoi gwybod i chi beth yw canlyniad ei adolygiad.
+pdf.ruling.section.askForReview.furtherInfo=I gael rhagor o wybodaeth am adolygiadau, darllenwch y dudalen ganlynol.
+pdf.ruling.section.askForReview.guidance.url=<a href="www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy" target="_blank">www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy</a>
+pdf.ruling.section.askForReview.requestReview=I wneud cais am adolygiad, e-bostiwch, neu ysgrifennwch at, Swyddfa'r Cyfreithiwr a Gwasanaethau Cyfreithiol CThEF (SOLS) yn uniongyrchol. Mae'r cyfeiriad e-bost a'r cyfeiriad post i'w gweld isod.
+pdf.ruling.section.askForReview.solsTeam=Gan fod y tîm SOLS yn rhan wahanol o CThEF, bydd angen yr wybodaeth ganlynol arnynt i gael hyd i'ch achos ac i ymgymryd â'r adolygiad:
```

Comment:
Apostrophes are not displayed currently. Please use the curved one for consistency in welsh. Update to `i’ch` and `â’r` thanks